PerformanceTest_Cycling_Report.qmd

```diff
@@ -541,78 +538,154 @@ instDorsalMod <- brm(data = staticDat_instPres, # Bayes model
 #| echo: false
 #| warning: false
 #| layout-ncol: 2 
+
+
 p <- withinSubPlot(staticDat_instPres , colName = 'instepDorsalPressure', dir = 'higher')
-p + ylab('Ave. Instep Pressure (kPa)')  
+p + ylab('Avg. Instep Pressure (kPa)')
 
-extractVals(staticDat_instPres, instDorsalMod, otherConfigs, baseline, 'instepDorsalPressure', 'higher') 
+extractVals(staticDat_instPres, instDorsalMod, otherConfigs, baseline, 'instepDorsalPressure', 'higher')
 ```
 
-### Peak Toe Pressure
+#\| echo: false
+
+#\| warning: false
+
+#\| layout-ncol: 2
+
+p \<- withinSubPlot(staticDat_MAXinstPres , colName = 'maxDorsalPressure', dir = 'lower')
+
+p + ylab('Peak Instep Pressure (kPa)')
+
+extractVals(staticDat_MAXinstPres, MAXinstDorsalMod, otherConfigs, baseline, 'maxDorsalPressure', 'lower')
 
 ```{r}
 #| echo: false
 #| warning: false
-#| include: false 
+#| include: false
 
-# Plantar Max Toe Pressure
 
-toemaxDat<- pressDat %>% 
+staticDat_MAXinstPres <- staticDat %>%
   group_by(Subject) %>%
-  mutate(z_score = scale(maxmaxToes)) %>% 
+  mutate(z_score = scale(maxDorsalPressure)) %>%
   group_by(Config)
 
 
+# Look at histogram of the data if any outliers need to be removed
+ggplot(data = staticDat_MAXinstPres, aes(x = maxDorsalPressure, fill = Config)) + geom_histogram() + facet_wrap(~Subject)
 
 
-toemaxMod <- brm(data = toemaxDat, # Bayes model
+
+MAXinstDorsalMod <- brm(data = staticDat_MAXinstPres, # Bayes model
               family = gaussian,
-              z_score ~ Config + (1 + Config| Subject), #fixed effect of configuration and time period with a different intercept and slope for each subject
+              z_score ~ Config + Trial + (1 | Subject), #fixed effect of configuration and time period with a different intercept and slope for each subject
```

Comment:
update comment.. for independent intercepts per subject but not config.. due to limited data